assets/component-cart-items.css

```diff
@@ -201,12 +213,13 @@ cart-remove-button .icon-remove {
   }
 
   .cart-item__quantity {
-    grid-column: 2 / 4;
+    grid-column: 2 / 5;
+    flex-wrap: wrap;
   }
```

Comment:
Im not too sure I understand what this should be aligned too 🤔  I don't dislike how it is right now but confused by `Align left the Quantity column with the Subtotal and Checkout button to the left` 
[screenshot](https://screenshot.click/21-08-1hbvu-kl4fu.png)